Zapper-fi/studio
# fix(curve): add cadc/usdc pool and farm

## Description



I would like to see my cadc/usdc curve position: https://curve.fi/factory-crypto/12

This PR adds the relevant pool and gauge farm. It includes a very minor refactor in the ethereum farm position fetcher: to use the app definition governance token address (CRV) as reward address instead of a hard coded address for CRV.

## Checklist

- [x] (optional) As a contributor, my Ethereum address/ENS is: agamemnon.eth

## How to test?



Make sure the farm is found and built properly (search the result for CADC)
`GET http://localhost:5001/apps/curve/positions?groupIds[]=farm`

Make sure the balance is accurate
`GET http://localhost:5001/apps/curve/balances?addresses[]=0x2BA829474AC766261633375eE002B2B75e775FdB`
